using-mobile-extensions/adobe-analytics/analytics-faq.md

```diff
@@ -16,6 +16,14 @@ If you see SDK identifiers unexpectedly change, try the following fixes to addre
 
 ----
 
+### What identifier types does the AEP Mobile SDK use for Analytics implementations
+
+The Analytics mobile extension uses the ECID as the main identifier type for mobile visitors,and so it requires the Identity extension to operate. The Identity extension preserves the ECID for current visitor, respectively generates a new ECID when the application is first installed. For more details, see the [Identity extension](../../foundation-extensions/mobile-core/identity) documentation.
```

Comment:
No need for the and here. I'd just go with: "... for mobile visitors, so it requires the..."